script/translations_download

```diff
@@ -25,23 +25,23 @@ declare -A PROJECT_ID=( \
     [backend]="130246255a974bd3b5e8a1.51616605" \
     )
 
+if [[ $(uname -s) == Linux* ]] ; then
+    curl -sfL https://raw.githubusercontent.com/lokalise/lokalise-cli-2-go/master/install.sh | sh
```

Comment:
This does impose a security risk, as we are running third-party scripts from external unisolated.

Reply:
Why not use their [node client](https://github.com/lokalise/node-lokalise-api) instead?  Could then integrate the download into the gulp pipeline.